pygrocy/data_models/product.py

```diff
@@ -47,6 +70,9 @@ def _init_empty(self):
         self._available_amount = None
         self._best_before_date = None
 
+        self._default_quantity_unit_purchase = None
```

Comment:
Could you add basic test assertions [over here](https://github.com/SebRut/pygrocy/blob/ffcb306a808ea4b202adc54985d33ad5d9eaa5c0/test/test_product.py#L25) for these fields please?

Reply:
Done. I have added some assertions (and forgot to commit file ;-))